packages/trip-form/src/index.ts

```diff
@@ -8,19 +8,29 @@ import SettingsSelectorPanel from "./SettingsSelectorPanel";
 import SliderSelector from "./SliderSelector";
 import * as Styled from "./styled";
 import SubmodeSelector from "./SubmodeSelector";
+import MetroModeSelector from "./MetroModeSelector";
+import {
+  getActivatedModesFromQueryParams,
+  useModeState
+} from "./MetroModeSelector/utils";
 import TripOptions, { Styled as TripOptionsStyled } from "./TripOptions";
+import defaultModeSettings from "../modeSettings.yml";
 
 export {
   CheckboxSelector,
   DateTimeSelector,
+  defaultModeSettings,
   DropdownSelector,
   GeneralSettingsPanel,
+  getActivatedModesFromQueryParams,
+  MetroModeSelector,
   ModeButton,
   ModeSelector,
   SettingsSelectorPanel,
```

Comment:
Is this actually backwards compatible with the `SettingSelectorPanel`? If not we can probably remove it

Reply:
Well, the OTP-UI PR is fully backwards compatible, it's OTP-RR side that breaks stuff. I think we can remove this in a future breaking change.